psi4/driver/qcdb/molecule.py

```diff
@@ -1314,7 +1314,12 @@ def run_dftd3(self, func=None, dashlvl=None, dashparam=None, dertype=None, verbo
         # hack as not checking type GRAD
         for k, qca in jobrec['extras']['qcvars'].items():
             if isinstance(qca, (list, np.ndarray)):
-                jobrec['extras']['qcvars'][k] = np.array(qca).reshape(-1, 3)
+                # Coerce gradient into proper shape
+                try:
+                    jobrec['extras']['qcvars'][k] = np.array(qca).reshape(-1, 3)
```

Comment:
it's not well broadcast yet, but make sure you have `msgpack-python` installed. it is/will be a req'd dep for psi4 (optional dep of qcel) and will mean much less reshaping.